tools/rust_analyzer/main.rs

```diff
@@ -148,13 +108,10 @@ struct Config {
     #[structopt(long)]
     execution_root: Option<PathBuf>,
 
-    // If not specified, uses the result of `bazel info bazel-bin`.
-    #[structopt(long)]
-    bazel_bin: Option<PathBuf>,
-
     #[structopt(long, default_value = "bazel")]
     bazel: PathBuf,
 
-    #[structopt(long, default_value = "//:rust_analyzer")]
-    bazel_analyzer_target: String,
+    /// Comma-separated list of target patterns
```

Comment:
I'd keep the flag syntax here identical to bazel - "space separated" list of target patterns without a dedicated --flag.

Reply:
Are you familiar enough with [StructOpt](https://crates.io/crates/structopt) to know how to do that easily?

Reply:
I think the idea being proposed is to use the final varargs to do that, and it looks like you might be able to do that by specifying neither short nor long in the annotation, like this from the example:
```
    /// Files to process
    #[structopt(name = "FILE", parse(from_os_str))]
    files: Vec<PathBuf>,
```